index.html

```diff
@@ -1838,15 +1849,18 @@ <h3>
     </section>
 
     <section>
-      <h3>The <dfn>PropertyMap</dfn> type</h3>
+      <h3>The <dfn>PropertyReadMap</dfn> type</h3>
       <p>
         Represents a map of <a>Property</a> names as strings to a value that the <a>Property</a> can take. It is used as a property bag for interactions that involve multiple <a>Properties</a> at once.
```

Comment:
I am not sure that _value_ is the right term here. With the current API model, we return a map of strings and InteractionOutput which is a _gateway_ to the real value. Should we rephrase to something like this:

" Represents a map of Property names as strings to an InteractionOutput that _can be used to retrieve_/_contains_ the value of Property"?